VideoWeb/VideoWeb/ClientApp/src/app/shared/pipes/ngx-date.pipe.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { NgxDatePipe } from './ngx-date.pipe';
+import { translateServiceSpy } from 'src/app/testing/mocks/mock-translation.service';
+
+describe('DatePipe', () => {
+    const translateService = translateServiceSpy;
+    const testDate = new Date('2021-06-16');
+    const testFormat = 'TestFormat';
+
+    let pipe: NgxDatePipe;
+    const expectedValue = 'expectedValue';
+
+    beforeEach(() => {
+        translateService.instant.calls.reset();
+        pipe = new NgxDatePipe(translateService);
+        spyOn(NgxDatePipe.prototype, 'transform').and.returnValue('expectedValue');
```

Comment:
This should be DatePipe. We use spyOn to prevent it from calling code external to what we want to test. This is overriding the function we want to test which is why the coverage is failing.